src/pages/Home/FriendReadonlyMap.js

```diff
@@ -95,6 +95,16 @@ const FriendReadonlyMap = () => {
     handleTripData(data);
     handleLoaded(true);
   }
+  function geoscoreinfo() {
```

Comment:
Seems like this function is setting up the swal alert. Should it be named like so? Also for JS, the naming convention should be camelCase right?

Reply:
yep, will change to geoScoreSwal